src/components/pages/Home/MainHeader.js

```diff
@@ -32,15 +59,25 @@ export default function MainHeader(props) {
       </Menu.Item>
     </Menu>
   );
+
   return (
     <div className="site-page-header">
       <img src={HRFLogo} alt="HRF Logo" />
       <div id="nav-flex-container">
         <UploadCase getPendingCases={getPendingCases} />
+        <Notifications
+          notifications={MOCK_NOTIFICATIONS}
+          onClickView={onClickViewNotification}
+          onClickDismiss={onClickDismissNotification}
+        />
         <Dropdown className="account-drop-down" overlay={menu}>
           <ul className="accounts" onClick={e => e.preventDefault()}>
```

Comment:
The event indicator here doesn't match the rest of the included code. ie:   e or evt